Sources/Sentry/SentryClient.m

```diff
@@ -1120,6 +1122,23 @@ - (void)removeAttachmentProcessor:(id<SentryClientAttachmentProcessor>)attachmen
     return processedAttachments;
 }
 
+- (void)captureLog:(SentryLog *)log
+{
+    [self.logBatcher processLog:log with:[[SentryScope alloc] init]];
+}
+
+- (void)captureLog:(SentryLog *)log withScope:(SentryScope *)scope
+{
+    [self.logBatcher processLog:log with:scope];
+}
+
+// SentryLogBatcherDelegate
```

Comment:
Using the delegation pattern to avoid injecting `transportAdapter` into the batcher.